composer.json

```diff
@@ -15,31 +15,27 @@
         }
     },
     "require": {
-        "php": ">=7.2,<7.5",
+        "php": "^7.2 || ^8.0",
         "ext-json": "*",
         "graylog2/gelf-php": "^1.4.2",
         "monolog/monolog": "^1.24 || ^2.0",
-        "sentry/sentry-symfony": "^3.0",
+        "sentry/sentry-symfony": "^4.0",
         "symfony/framework-bundle": "^3.4.26|^4.2.7",
         "symfony/monolog-bundle": "^3.4"
     },
     "require-dev": {
-        "doctrine/doctrine-bundle": "^1.4",
-        "doctrine/orm": "^2.5.14",
-        "doctrine/persistence": "^1.3",
-        "mockery/mockery": "^1.2",
-        "paysera/lib-php-cs-fixer-config": "^2.3",
-        "phpunit/phpunit": "^6.5",
-        "symfony/yaml": "^3.4.34|^4.3"
+        "doctrine/annotations": "^1.14",
+        "doctrine/doctrine-bundle": "~2.6.0",
+        "doctrine/orm": "^2.10",
```

Comment:
We will need to add some of Doctrine packages as non-dev one, because we have references to them in [FormatterTrait](src/Service/Formatter/FormatterTrait.php).

Actually we have other soft dependencies except Doctrine.

Reply:
There is no need to make Doctrine packages mandatory. FormatterTrait tries to deal with them if they are installed but they're not mandatory for it to work. The code first checks for instances of the Doctrine classes before actually using them.